dspace/src/main/docker-compose/eclipse-debug.yml

```diff
@@ -0,0 +1,25 @@
+#
+# The contents of this file are subject to the license and copyright
+# detailed in the LICENSE and NOTICE files at the root of the source
+# tree and available online at
+#
+# http://www.dspace.org/license/
+#
+
+version: "3.7"
+
+services:
+  dspace:
+    ports:
+    - published: 8000
+      target: 8000
+    environment:
+      JPDA_ADDRESS: 8000
+      JPDA_TRANSPORT: dt_socket
+    entrypoint:
+    - /bin/bash
+    - '-c'
+    - |
+      /dspace/bin/dspace database migrate
```

Comment:
Just an FYI, this simple `database migrate` command is not enough to ensure the database is "ready" before Tomcat starts.  See https://github.com/DSpace/DSpace/pull/2635